src/main/java/uk/co/ramp/policy/ContactIsolationPolicy.java

```diff
@@ -0,0 +1,32 @@
+package uk.co.ramp.policy;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import uk.co.ramp.io.DiseaseProperties;
+import uk.co.ramp.people.Case;
+
+class ContactIsolationPolicy implements IsolationPolicy {
+    private static final Logger LOGGER = LogManager.getLogger(ContactIsolationPolicy.class);
+
+    private final SingleCaseIsolationPolicy singleCaseIsolationPolicy;
+    private final DiseaseProperties diseaseProperties;
+
+    ContactIsolationPolicy(SingleCaseIsolationPolicy singleCaseIsolationPolicy, DiseaseProperties diseaseProperties) {
+        this.singleCaseIsolationPolicy = singleCaseIsolationPolicy;
+        this.diseaseProperties = diseaseProperties;
+    }
+
+    @Override
+    public boolean isContactIsolated(Case caseA, Case caseB, double contactWeight, double actualInfectedProportion, int currentTime) {
+        boolean isCaseAInIsolation = singleCaseIsolationPolicy.isIndividualInIsolation(caseA.id(), caseA.status(), caseA.alertStatus(), caseA.compliance(), actualInfectedProportion, currentTime);
+        boolean isCaseBInIsolation = singleCaseIsolationPolicy.isIndividualInIsolation(caseB.id(), caseA.status(), caseA.alertStatus(), caseA.compliance(), actualInfectedProportion, currentTime);
```

Comment:
should this line reference caseA? or should it be:
`
isIndividualInIsolation(caseB.id(), caseB.status(), caseB.alertStatus(), caseB.compliance(), actualInfectedProportion, currentTime);
`

Reply:
You're right. Copy paste error . I'll make the change as you suggested and additional unit tests

Reply:
I couldn't make out if I was being dense or not.

Reply:
It's a good spot!